packages/drivers/odsp-driver/src/odspDriverUrlResolverForShareLink.ts

```diff
@@ -137,7 +137,7 @@ export class OdspDriverUrlResolverForShareLink implements IUrlResolver {
         const odspResolvedUrl = await new OdspDriverUrlResolver().resolve(requestToBeResolved);
 
         if (isSharingLinkToRedeem) {
-            odspResolvedUrl.sharingLinkToRedeem = request.url.split("?")[0];
```

Comment:
Is this intentional change? Do we pass query param to redeem process?

Reply:
Yes. We do need the whole link for making a redeem call as I figured out. I have asked William, that it should not affect the other single RT redeem during trees/latest even if we send the query params.